src/fermilib/utils/_operator_utils.py

```diff
@@ -153,8 +153,8 @@ def save_operator(operator, file_name=None, data_directory=None):
 
     tm = operator.terms
     with open(file_path, 'wb') as f:
-        marshal.dump((operator_type, dict(izip(tm.iterkeys(),
-                                          imap(complex, tm.itervalues())))), f)
+        marshal.dump((operator_type, dict(zip(tm.keys(),
```

Comment:
I'm not sure actually - I actually thought that Python3's d.keys() was 2's d.iterkeys(). In fact I didn't know about viewkeys at all. It looks like both should work, but I know that future.utils.iterkeys passes tests for both 2 and 3 - it would be interesting to see if there's anything different with viewkeys, though I don't think we need any of its special properties like dynamic view.